src/bin/stress-test/worker-routine.ts

```diff
@@ -165,7 +159,7 @@ const checker = async (
   config: WorkerConfig,
   syncClients: SyncClient[]
 ): Promise<void> => {
-  await delay((60 / config.repoReadsPerMin) * 1000)
+  await delay(config.repoCheckDelayInSeconds)
```

Comment:
There is a factor of 1000 difference between the old and new params.
Is that intentionally?

Reply:
That's not intentional... That's a bug.